components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -657,12 +657,13 @@ const ContributorHighlightCard = ({
               direction="top"
               content={`${getEmojiReactors(reaction_users)} reacted with ${getEmojiNameById(emoji_id)} emoji`}
             >
-              <div
+              <button
                 className={`px-1 py-0 md:py-0.5 hover:bg-light-slate-6 transition  md:px-1.5 shrink-0 border flex items-center justify-center rounded-full cursor-pointer ${
                   isUserReaction(emoji_id) && "bg-light-slate-6"
                 }`}
                 onClick={async () => (sessionToken ? handleUpdateReaction(emoji_id) : signIn({ provider: "github" }))}
               >
+        </button>
```

Comment:
hey @CBID2! You just need to remove this `</button>` AFTER you replace `</div> `with `</button>` on line `671`.

Reply:
```suggestion

```